EIPS/eip-6800.md

```diff
@@ -157,25 +154,26 @@ def get_tree_key(address: Address32, tree_index: int, sub_index: int):
         bytes([sub_index])
     )
     
-def get_tree_key_for_version(address: Address32):
-    return get_tree_key(address, 0, VERSION_LEAF_KEY)
-    
-def get_tree_key_for_balance(address: Address32):
-    return get_tree_key(address, 0, BALANCE_LEAF_KEY)
-    
-def get_tree_key_for_nonce(address: Address32):
-    return get_tree_key(address, 0, NONCE_LEAF_KEY)
+def get_tree_key_for_basic_data(address: Address32):
+    return get_tree_key(address, 0, BASIC_DATA_LEAF_KEY)
 
 # Backwards compatibility for EXTCODEHASH    
 def get_tree_key_for_code_keccak(address: Address32):
     return get_tree_key(address, 0, CODE_KECCAK_LEAF_KEY)
-    
-# Backwards compatibility for EXTCODESIZE
-def get_tree_key_for_code_size(address: Address32):
-    return get_tree_key(address, 0, CODE_SIZE_LEAF_KEY)
 ```
 
-When any account header field is set, the `version` is also set to zero. The `code_keccak` and `code_size` fields are set upon contract creation.
+An account's `version`, `balance`, `nonce` and `code_size` fields are packed in the value found at `BASIC_DATA_LEAF_KEY`:
+
+| Name        | Offset | Size |
+| ----------- | ------ | ---- |
+| `version`   | 0      | 1    |
+| `nonce`     | 4      | 8    |
+| `code_size` | 12     | 4    |
+| `balance`   | 16     | 16   |
+
+Bytes `1..3` are reserved for future use.
+
+When any account header field is set, the `version` field is also set to zero. The `code_keccak` and `code_size` fields are set upon contract or EoA creation.
```

Comment:
should we also add a section about 7702 (eoa temporary contract code) accesses?

Reply:
In this PR already added rules for `AUTHCALL`. So mentioning 7702 would be confusing?

Maybe it's better to not yet assume 3074 nor 7702 in any part of the PR (i.e: remove the existing `AUTHCALL` mentioning) and add a TODO saying `Add 7702/3074 to the spec`.